### ISSUE TITLE
Can't display temperature, calories, distance on custom watchface using Gadget Bridge

### ISSUE THREAD

**Issue by Coincoin35** [Freeyourgadget/Gadgetbridge#1664]
Hi
it is more a question (but can't find a forum to discuss about gadget bridge features):
I installed easily (following crystal-clear wiki instructions) new watchfaces on my Bip to get on one screen weather, temperature, time, date, number of steps, distance, estimated calories burned. 
- Temperature, though accessible on the Bip menu and on weather app, doesn't display on the watchface.
- I see number of steps but distance and estimated calories burned remain to zero. 

Do I need to configure something that I would not have seen on the menus? 

Thanks for incoming help (that I would try to introduce in the wiki if solved)

**Comment by vanous** [comment]
@Coincoin35 welcome.

Sounds like not a very good watchface. Stock watchfaces show you all info? (like the complex analog one?)
Feel free to join us on matrix: https://riot.im/app/#/room/#gadgetbridge:matrix.org

**Comment by Coincoin35** [comment]
you right. I can see temperature but not calories and distance.

**Comment by vanous** [comment]
Did you set personal info in the "About you"? Height, weight and probably age might be needed for some of these calculations.

**Comment by Coincoin35** [comment]
Yes I did it.

**Comment by vanous** [comment]
Have you had the watch for multiple days? How many steps do you have today?

**Comment by ashimokawa** [comment]
Which watch face?

**Comment by vanous** [comment]
I would suggest testing on a stock watchface, this one works: 
<img src="https://user-images.githubusercontent.com/3680926/66228082-e031d080-e6de-11e9-917a-dc6799eb559d.jpg" width="200">

**Comment by Coincoin35** [comment]
First, I switched to the watchface you proposed, with 5200 steps, after 10 minutes it still displayed 0 calories. But after one hour of some activities, I have now 9013 steps and now it shows 180 calories. 

Summary: So it seems I used bad watchfaces: this one  [this one](https://amazfitwatchfaces.com/bip/resource/img/5c33a80b21fc9_07012019.gif), [this one](https://amazfitwatchfaces.com/bip/resource/img/5c7014a87adba_22022019.gif) and this [one](https://amazfitwatchfaces.com/bip/resource/img/5c13d50591702_14122018.gif)
With the stock one (they should use name so we talk about them!), I had to wait for nearly one hour to have a calorie actualization.  
Thanks for all your help Vanous!
   
Status: Solved. 
Hope that I can help one day beyond my financial support. Best

**Comment by Coincoin35** [comment]
PS:
...Weird! I just try back the bad watchfaces: Now those ones that didn't display calories and distance display them ! XD
But still not the temperature...
(I can take photo but i don't know how to post them.)

**Comment by vanous** [comment]
Temperature comes and goes. I think there is some internal timeout and if temperature isn't refreshed, the watch system stops providing it.
So you must refresh and look. I have seen watchfaces which didn't work or so I though...

Same for the whole weather forecast. On MB3, weather lasts one day (although being multiday forecast) .

**Comment by IzzySoft** [comment]
And for the heart beat. Unless you're dead. At least going by your screenshot :speak_no_evil:

**Comment by vanous** [comment]
Let me check...

**Comment by ashimokawa** [comment]
@Coincoin35 
Financial support? Donations are not working AFAIK. :(

**Comment by Coincoin35** [comment]
so can't trust here? :'(
https://liberapay.com/Gadgetbridge

**Comment by IzzySoft** [comment]
No privacy friendly options there anymore, they just offer Paypal (:nauseated_face:) and Stripe (:mask:) – which is why many privacy-focused creators (including Gadgetbridge, including me) cannot withdraw or even closed their account with them. They cannot offer e.g. SEPA payment (which would require a banking license) – and refuse offering support for e.g. Bitcoin. Further: donations received via Paypal can only be retrieved via Paypal, etc.

TL;DR: Liberapay became useless. I'm just using my profile their to point out where donations can be made (Bitcoin, for more see {my-donation-page}).

BTW: they stopped having my trust at the moment they *ignored* my privacy concerns, saying something like "ah, cannot be that bad, just try it you'll like it". Nope, sorry.

**Comment by ashimokawa** [comment]
@CoinCoin35 you can but I think donation do not work or did they?

**Comment by vanous** [comment]
Hmm, even Codeberg uses librepay, thus Stripe/PayPal only... so no other options?

**Comment by ashimokawa** [comment]
Codeberg has a PayPal account, yes. Unfortunately it had to be done.

**Comment by IzzySoft** [comment]
Whut, you fell? :scream: And have chosen the biggest of the devils… I suppose you took the 2 days required to get through their AGBs, have you?

**Comment by vanous** [comment]
Hi all,

the team has decided to proceed with Librepay again, see copy/paste of the announcement posted in Matrix:

> After more than a year we bit the bullet and re-enbled donations on liberapay by complying with what had to be done to make that work. We were already almost dried up and were unable buy devices we wanted to support. To make Gadgetbridge even better please consider to donate.

>Do not want to push anyone but it was requested over and over again. And supporting new devices without having them is very time consuming ;)

So thank you very much for proceeding if you still consider supporting the project financially! 

<a href="https://liberapay.com/Gadgetbridge/donate" rel="nofollow"><img src="https://camo.githubusercontent.com/ae8af018f80649f3d379eb23dbf59acceaffa24e/68747470733a2f2f6c69626572617061792e636f6d2f6173736574732f776964676574732f646f6e6174652e737667" alt="Donate" data-canonical-src="https://liberapay.com/assets/widgets/donate.svg" style="max-width:100%;"></a>
